functions/unittest/parseAirtable.test.js

```diff
@@ -0,0 +1,70 @@
+const mockAirtable = require ('../airtable-shared-view.js');
+const { parseAirtableData } = require('../airtable-shared-view.js');
+
+describe('mapColumn() should detect any ftm-name in [] and set up the appropriate translator based on column type', () => {
+  const testColumn = [
+    {
+      name: 'This string should not be matched [matched-string]',
+      type: 'text',
+      id: 'first',
+    },
+    {
+      name: 'Nothing should match here',
+      type: 'select',
+      id: 'second',
+    },
+  ];  
+  test('If the column name has a string encapsulated in [], regex should match and a ftm-name property should be made', () => {
+    const copyTestColumn = JSON.parse(JSON.stringify(testColumn));
```

Comment:
might need babel to do this but another way to duplicate an object is:

```
const copyTestColumn = { ...testColumn }
```

Without babel I think this would work:
```
const copyTestColumn = Object.assign({}, testColumn);
```

parsing + stringifying works but it's a little less wonky

Reply:
Nice! I only did a brief search about cloning JS objects and the parsing+stringifying method was the first one that showed up so I just went with this. I'm not too sure if we installed the babel-jest extension but I'll try the Object.asisgn()